### PR TITLE
Set up basic componenet and dataflow

### PR DESCRIPTION
**Pull request by tmikeschu**
@DavidKnott @ski-climb @ethanbennett @case-eee ready for review

This sets up a basic shell for the component and data flow of the react client. I use a fixture to simulate the shape of the data we'll work with once we have the back-end sorted out, and it will be easy to adjust once that is the case.

screen shot 2017-03-29 at 7 56 56 am

Add Travis CI badge to README
Add reset.scss
Develop Header, ProductPath, and Location components
Work on dataflow for locations
Update styling and directory structure
Update tests to include props
[Finishes #142557989]

### CODE REVIEW

File: src/App/Header/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { shallow } from 'enzyme';
+import Header from './Header';
+
+describe('<Header />', () => {
+  it('renders without crashing', () => {
+    const header = shallow(<Header />); 
+    expect(header).toBeTruthy();
+  });
+
+  it('has a Sourcery heading', () => {
```
Comment (case-eee) [inline src/App/Header/Header.test.js]: 👍

**Comment by case-eee** [comment]
love all the testing @tmikeschu 👍

**Comment by tmikeschu** [comment]
I'm going to add tests for the App component rendering the ProductPath

**Comment by DavidKnott** [comment]
@tmikeschu Great job testing and getting started on the data flow! Looks good to merge.

**Comment by tmikeschu** [comment]
@ski-climb thanks for your comments. I have some wireframes now that I think answer these questions, let's discuss this afternoon!

**Pull request merged**